sat/sat.go

```diff
@@ -99,15 +108,11 @@ func New(config *Config) (*Sat, error) {
 // Start starts Sat's Vektor server and Grav discovery
 func (s *Sat) Start() error {
 	// start Vektor first so that the server is started up before Grav starts discovery
-	go func() {
-		if err := s.v.Start(); err != nil {
-			if err == http.ErrServerClosed {
-				// that's fine, do nothing
-			} else {
-				log.Fatal(errors.Wrap(err, "failed to Start server"))
-			}
+	if err := s.v.Start(); err != nil {
```

Comment:
This needs to be in a goroutine because it blocks.. everything afterwards needs to happen too :)

Reply:
Fixed in d1fda89c